src/core/caniuse.js

```diff
@@ -60,18 +48,29 @@ export async function run(conf) {
   if (!options.feature) {
     return; // no feature to show
   }
-  const featureURL = new URL(options.feature, "https://caniuse.com/").href;
 
-  const caniuseCss = await caniuseCssPromise;
+  const caniuseCss = await loadStyle();
   document.head.appendChild(html`<style class="removeOnSave">
     ${caniuseCss}
   </style>`);
 
+  const apiUrl = options.apiURL || API_URL;
+  // Initiate a fetch, but do not wait. Try to fill the cache early instead.
+  conf.state[name] = {
+    fetchPromise: fetchStats(apiUrl, options),
+  };
+}
+
+export async function run(conf) {
+  const options = conf.caniuse;
+  if (!options || !options.feature) return;
```

Comment:
```suggestion
  if (!options?.feature) return;
```

Reply:
OMG are we ready for optional chaining now?! :star_struck:

Reply:
Optional chain all the things!! https://caniuse.com/mdn-javascript_operators_optional_chaining

Reply:
Wait. Should we wait for Samsung Internet?

Reply:
That's a good question... I think the data is just outdated and it's actually supported. Filed https://github.com/mdn/browser-compat-data/issues/9146

Reply:
Yeah, let's start using optional chaining.